src/java.base/share/classes/java/lang/Class.java

```diff
@@ -140,22 +140,24 @@
  * }}
  *
  * It is also possible to get the {@code Class} object for a named
- * class or interface (or for {@code void}) using a <i>class literal</i>.
+ * class or interface (or for {@code void}) using a <dfn>class literal</dfn>
+ * (JLS  {@jls 15.8.2}).
```

Comment:
```suggestion
 * (JLS {@jls 15.8.2}).
```
Also, the preexisting description of "class or interface" as represented by `java.lang.Class` is incorrect; it also represents primitive types, including void, and array types. But this problem exists in the first-sentence summary as well, so I believe we should fix it in another RFE.